content/contributing-code/github-repo-guidelines/contents.lr

```diff
@@ -24,57 +57,52 @@ ready for external contributors.
 - `/.github/CODEOWNERS`: Defined code owners
   - [About code owners - GitHub Help][about-owners]
 - `/.cc-metadata.yml`: The standard [CC metadata YAML file](#cc-metadata-file).
-- `/CODE_OF_CONDUCT.md`: Code of Conduct
-  - Feel free to use our standard [`CODE_OF_CONDUCT.md`][conduct] file from
-    [creativecommons/.github][dot-github]
-    - [Adding a code of conduct to your project - GitHub Help][help-condcut]
 - `/LICENSE`: license file.
-- `/CONTRIBUTING.md`: contributor guidelines
-  - Can be project-specific or our standard [`CONTRIBUTING.md`][contributing]
-    file from [creativecommons/.github][dot-github]
-  - [Setting guidelines for repository contributors - GitHub
-    Help][setting-guidelines]
-- `/README.md`: read me information file
+- `/README.md`: repository information file
+  - begin with an H1 heading identical to repository name followed by a
+    repository description
   - Must documents how to install and build the project locally and documents a
```

Comment:
Not a part of this change, but `documents` should be `document` here if you want to bundle that fix in wit this PR.

Reply:
I corrected it.